packages/neuron-wallet/src/database/chain/ormconfig.ts

```diff
@@ -33,17 +33,18 @@ import { RemoveAssetAccountWalletID1589273902050 } from './migrations/1589273902
 
 export const CONNECTION_NOT_FOUND_NAME = 'ConnectionNotFoundError'
 
-const dbPath = (name: string): string => {
+export const dbPath = (name: string): string => {
```

Comment:
I guess these changes are mainly meant for the convenience of your test suit in this PR. 

Let's try to resolve the issue in the context of the test suit and **avoid** these changes in this file.

Reply:
> I guess these changes are mainly meant for the convenience of your test suit in this PR.
> 
> Let's try to resolve the issue in the context of the test suit and **avoid** these changes in this file.

The fact is that the old one is misleading. I expect the `dbPath` to return the real dbPath but it returns `cell-0x1234.sqlite` instead of `:memory:` in tests. Because of its value is discarded in the test: https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/src/database/chain/ormconfig.ts#L43

Reply:
And that why I've to use 
```
const connection = getConnection()
const dbPath = connection.options.database as string`
```
in the code.